extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+/**
+ * Discussion user log page
+ */
+class SpecialDiscussionsLogController extends WikiaSpecialPageController {
+	// @todo
+	//1. Pagination
+	//2. Add date range input field
+
+	const KIBANA_BASEURL = 'http://query.es.service.sjc.consul:9200';
+	const PAGINATION_SIZE = 50;
+	const DAYS_RANGE = 15;
+	const HTTP_STATUS_OK = 200;
+	const NO_USER_MATCH_ERROR = "Provided username did not match any user";
```

Comment:
Well, I was thinking about it, but this const is use just as an exception message, it's not user-facing. I assumed it makes more sense to always have it in one form, so there's an easier way to construct a query to find these in Kibana etc. What do you think?